ui/src/app/shared/services/parser.spec.ts

```diff
@@ -14,6 +14,9 @@ describe('parser:', () => {
     line = parseResult.lines[0];
     expect(line.errors).toBeNull();
     node = line.nodes[0];
+    // [{"group":"UNKNOWN_0","type":"sink","name":"time","options":{},"optionsranges":{},
```

Comment:
May want to JSDoc these so that you can explain the JSON.

Reply:
Might be better to create a full expected mock object somewhere else and then assert against that. I really don't see this very useful to put some long examples in comments and then assert some fields from it. Also makes code to look not that pretty.

Reply:
Comments in tests reworked into asserts.